web/views/error_view.ex

```diff
@@ -9,6 +9,14 @@ defmodule Org.ErrorView do
     "Internal server error"
   end
 
+  def render("401.json", _assigns) do
+    %{error: "Please sign in"}
```

Comment:
I would rather stick with the generic/spec HTTP status messages, i.e. "Unauthorized" and "Forbidden" (below). As a general rule, I prefer not to be very specific about what is wrong (any more than necessary), for security reasons.